Source/PLCrashLogWriter.m

```diff
@@ -430,7 +426,15 @@ plcrash_error_t plcrash_log_writer_init (plcrash_log_writer_t *writer,
 
 #if TARGET_OS_IPHONE
     /* iPhone OS */
-    writer->system_info.version = strdup([[[UIDevice currentDevice] systemVersion] UTF8String]);
+    {
+        NSOperatingSystemVersion systemVersion = [NSProcessInfo processInfo].operatingSystemVersion;
```

Comment:
It's available only since iOS 8, that's probably fine, but doesn't match with current build settings.
See https://developer.apple.com/documentation/foundation/nsprocessinfo/1410906-operatingsystemversion?language=objc

It works now only because of
```
ld: warning: OS version (6.0.0) too small, changing to 8.0.0
```

I guess we can just set this setting to 8.0

Reply:
Fixed by https://github.com/microsoft/plcrashreporter/pull/38